source/Playnite/Localization/LocSource.xaml

```diff
@@ -222,6 +222,7 @@ Selecting "add to exclusion list" option will prevent games from being imported
     <sys:String x:Key="LOCProgressImportinEmulatedGames">Importing emulated games from {0}…</sys:String>
     <sys:String x:Key="LOCProgressLibraryGames">Downloading library updates…</sys:String>
     <sys:String x:Key="LOCProgressScanningGamesInstallSize">Scanning size of games in library…</sys:String>
+    <sys:String x:Key="LOCProgressScanningEmulatedGamesInstallSize">Scanning size of imported emulated games…</sys:String>

```

Comment:
I would leave out that "emulated" part. That way we can reuse this string for non-emulated games as well.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3100/commits/bfff4cae6d5ef7773c8616f178a170d5022f690f